contracts/orderbook/Keepers/OrderKeeper.sol

```diff
@@ -0,0 +1,48 @@
+pragma solidity ^0.8.0;
+import "../IOrderBook.sol";
+import "@openzeppelin-4.3.2/token/ERC20/extensions/IERC20Metadata.sol";
+import "../../governance/PausableGuardian_0_8.sol";
+
+contract OrderKeeper is PausableGuardian_0_8 {
+    IOrderBook public factory;
+
+    constructor(IOrderBook factoryAddress) {
```

Comment:
order keeper probably going to be a proxy so we can easily upgrade and won't have to register thru chainlink again.
you will be having hard time calling constructor thru proxy, convert to `initialize onlyOwner`

Reply:
True. This is not a proxy at the moment so its fine but will convert to one